contracts/src/TeleporterTokenDestination.sol

```diff
@@ -34,6 +34,8 @@ abstract contract TeleporterTokenDestination is
     /// @notice The ERC20 token this contract uses to pay for Teleporter fees.
     address public immutable feeTokenAddress;
 
+    uint256 public constant SEND_TOKENS_REQUIRED_GAS = 300_000;
```

Comment:
Curious how much gas you've typically seen it require? Could be good to add a comment justifying the value.

Reply:
I forgot to add a TODO, but I'm deferring calculating actual values to #23 . For now these are brought from example `ERC20Bridge`, added the TODO